models/stg_recharge__order_line_item.sql

```diff
@@ -31,12 +31,12 @@ final as (
         quantity,
         grams,
         price,
-        property_charge_interval_unit_type as charge_interval_unit_type,
-        property_charge_interval_frequency as charge_interval_frequency,
-        property_shipping_interval_unit_type as shipping_interval_unit_type,
-        property_shipping_interval_frequency as shipping_interval_frequency,
-        property_title,
-        property_price,
+        property_charge_interval_unit_type as charge_interval_unit_type, 
+        property_charge_interval_frequency as charge_interval_frequency, 
+        property_shipping_interval_unit_type as shipping_interval_unit_type, 
+        property_shipping_interval_frequency as shipping_interval_frequency, 
+        property_title, 
+        property_price, 
```

Comment:
Same comment about these all being custom fields. We should let the variables handle these.

Reply:
Removed.